backend/src/main/java/com/constval/repook/RepookApplication.java

```diff
@@ -1,9 +1,12 @@
 package com.constval.repook;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = { SecurityAutoConfiguration.class })
```

Comment:
Why you had to exclude dis?

Reply:
Das war einer der versuche den Network Error zu fixen.